### ISSUE TITLE
Problems creating new project on Windows

### ISSUE THREAD

**Issue by carlosm27** [hanami/hanami#1275]
Hi, Happy New Year.

I'm trying Hanami for the first time and following this [guide](https://guides.hanamirb.org/v2.0/introduction/getting-started/). And every time I execute the command  `hanami new bookshelf`, it throws this error:

```
 hanami new bookshelf
Created bookshelf/
-> Within bookshelf/
Created .env
Created README.md
Created Gemfile
Created Rakefile
Created config.ru
Created config/app.rb
Created config/settings.rb
Created config/routes.rb
Created config/puma.rb
Created lib/tasks/.keep
Created lib/bookshelf/types.rb
Created app/actions/.keep
Created app/action.rb
Running Bundler install...
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/dry-files-1.0.1/lib/dry/files/file_system.rb:363:in `rescue in with_error_handling': No such file or directory -  install --jobs=8 --quiet --no-color (Dry::Files::IOError)
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/dry-files-1.0.1/lib/dry/files/file_system.rb:360:in `with_error_handling'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/dry-files-1.0.1/lib/dry/files/file_system.rb:189:in `chdir'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/dry-files-1.0.1/lib/dry/files.rb:148:in `chdir'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/hanami-cli-2.0.2/lib/hanami/cli/files.rb:40:in `chdir'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/hanami-cli-2.0.2/lib/hanami/cli/commands/gem/new.rb:51:in `call'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/dry-cli-1.0.0/lib/dry/cli.rb:116:in `perform_registry'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/dry-cli-1.0.0/lib/dry/cli.rb:65:in `call'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/hanami-cli-2.0.2/exe/hanami:11:in `<top (required)>'
        from C:/Ruby31-x64/bin/hanami:32:in `load'
        from C:/Ruby31-x64/bin/hanami:32:in `<main>'
C:/Ruby31-x64/lib/ruby/3.1.0/open3.rb:222:in `spawn': No such file or directory -  install --jobs=8 --quiet --no-color (Errno::ENOENT)
        from C:/Ruby31-x64/lib/ruby/3.1.0/open3.rb:222:in `popen_run'
        from C:/Ruby31-x64/lib/ruby/3.1.0/open3.rb:103:in `popen3'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/hanami-cli-2.0.2/lib/hanami/cli/system_call.rb:94:in `block in call'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/bundler-2.4.2/lib/bundler.rb:418:in `block in with_unbundled_env'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/bundler-2.4.2/lib/bundler.rb:664:in `with_env'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/bundler-2.4.2/lib/bundler.rb:418:in `with_unbundled_env'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/hanami-cli-2.0.2/lib/hanami/cli/system_call.rb:93:in `call'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/hanami-cli-2.0.2/lib/hanami/cli/bundler.rb:121:in `bundle'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/hanami-cli-2.0.2/lib/hanami/cli/bundler.rb:66:in `install'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/hanami-cli-2.0.2/lib/hanami/cli/bundler.rb:79:in `install!'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/hanami-cli-2.0.2/lib/hanami/cli/commands/gem/new.rb:57:in `block (2 levels) in call'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/hanami-cli-2.0.2/lib/hanami/cli/generators/gem/app.rb:27:in `call'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/hanami-cli-2.0.2/lib/hanami/cli/commands/gem/new.rb:52:in `block in call'
        from C:/Ruby31-x64/lib/ruby/3.1.0/fileutils.rb:139:in `chdir'
        from C:/Ruby31-x64/lib/ruby/3.1.0/fileutils.rb:139:in `cd'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/dry-files-1.0.1/lib/dry/files/file_system.rb:190:in `block in chdir'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/dry-files-1.0.1/lib/dry/files/file_system.rb:361:in `with_error_handling'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/dry-files-1.0.1/lib/dry/files/file_system.rb:189:in `chdir'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/dry-files-1.0.1/lib/dry/files.rb:148:in `chdir'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/hanami-cli-2.0.2/lib/hanami/cli/files.rb:40:in `chdir'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/hanami-cli-2.0.2/lib/hanami/cli/commands/gem/new.rb:51:in `call'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/dry-cli-1.0.0/lib/dry/cli.rb:116:in `perform_registry'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/dry-cli-1.0.0/lib/dry/cli.rb:65:in `call'
        from C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/hanami-cli-2.0.2/exe/hanami:11:in `<top (required)>'
        from C:/Ruby31-x64/bin/hanami:32:in `load'
        from C:/Ruby31-x64/bin/hanami:32:in `<main>'

```

OS: Windows 11
Ruby: 3.1
Hanami: 2.0.2

Have a good day.

**Comment by cllns** [comment]
Seems like this may be the same issue as https://github.com/dry-rb/dry-files/issues/15

**Comment by jodosha** [comment]
@carlosm27 https://github.com/hanami/cli/pull/72 should have fixed the problem.
The bug fix will be released with the next version of Hanami: 2.0.3.
